.github/workflows/deploy.yml

```diff
@@ -0,0 +1,18 @@
+
+name: Build
+on:
+  pull_request:
+    branches:
+      - master
+
+
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Build and push to deploy
+    runs-on: ubuntu-latest
+    steps:
+      - run: "curl -XPOST -u \"${{secrets.USERNAME}}:${{secrets.TOKEN}}\" -H \"Accept: application/vnd.github.everest-preview+json\"  -H \"Content-Type: application/json\" https://api.github.com/repos/multinet-app/multinet-deploy/dispatches --data '{\"event_type\":\"build_application\", \"client_payload\": {\"repo\": \"faux-app\"}}'"
```

Comment:
```suggestion
      - run: "curl -XPOST -u \"${{secrets.USERNAME}}:${{secrets.TOKEN}}\" -H \"Accept: application/vnd.github.everest-preview+json\"  -H \"Content-Type: application/json\" https://api.github.com/repos/multinet-app/multinet-deploy/dispatches --data '{\"event_type\":\"build_application\", \"client_payload\": {\"repo\": \"view-nodelink\"}}'"
```

Reply:
We must have noticed this at the same time. I updated it in 540b6cb

Reply:
Great minds :joy: